src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -804,6 +804,20 @@ public class SimpleEvents {
 				)
 				.since("INSERT VERSION");
 		}
+
+		if (Skript.classExists("org.bukkit.event.block.VaultDisplayItemEvent")) {
+			Skript.registerEvent("Vault Display Item", SimpleEvent.class, VaultDisplayItemEvent.class,
+					"vault display[ing] item")
+				.description("Called when a vault in a trial chamber is about to display an item.")
+				.examples(
+					"""
+						on vault display item:
+							set event-item to a netherite ingot	
```

Comment:
```suggestion
					on vault display item:
						set event-item to a netherite ingot	
```
Indentation matters in multiline strings; the ending """ determines the '0 indentation' line.